server/routes/shared/showReferralView.ts

```diff
@@ -74,7 +74,7 @@ export default class ShowReferralView {
     const html = `<strong>Referral changes saved</strong>
           <hr class="govuk-section-break govuk-section-break--s">
           <p>
-             The service provider has been notified of the changes to this referral.
+          The service provider and your responsible officer have been notified about the change to this referral.
```

Comment:
should this contain something like "if there is one"? it's not always possible to notify the responsible officer (we have about 200 instances/day where we can't find their RO or they don't have emails)